charts/keycloak/templates/keycloak_db.yaml

```
@@ -35,6 +48,16 @@ spec:
           value: /data/pgdata
         - name: POSTGRES_DB
           value: keycloak
+{{- if .Values.keycloak.pvc.enabled }}
+        volumeMounts:
+          - mountPath: /data/pgdata
+            subPath: pgdata
+            name: db-data
+      volumes:
+        - name: db-data
+          persistentVolumeClaim:
+            claimName: postgresql-pvc
```

```suggestion
            claimName: keycloak-postgresql-pvc
```

Related to the suggested change above.